Lesson3/src/Homework.java

```diff
@@ -0,0 +1,120 @@
+     * - if remainder of the division number by 5 is zero - print "bar" (example of number - 10)
+     * - if remainder of the division number by both 3 and 5 is zero 0  and - print "foobar" (example of number - 15)
+     */
+    public static void foobar(int number) {
+        if (number % 3 == 0 && number % 5 == 0) {
+            System.out.println("foobar");
+        } else if (number % 5 == 0) {
+            System.out.println("bar");
+        } else if (number % 3 == 0) {
+            System.out.println("foo");
+        }
+
+    }
+
+    /**
+     * Steps to implement:
+     * - Read two ints from console
+     * - Create a two-dimensional int array (use ints that you read from console height and width)
+     * - Fill array with random values (under 100)
+     * - Print in console matrix with given size, but:
+     * - If remainder of the division array element by 3 is zero - print "+" sign instead of array element value
+     * - If remainder of the division array element by 7 is zero - print "-" sign instead of array element value
+     * - Print "*" otherwise
+     * <p>
+     * Example:
+     * - Values from console -  2 and 3
+     * - Array will look like this (values be different because it's a random)
+     * 6 11 123
+     * 1 14 21
+     * - For this values output in console should be:
+     * <p>
+     * + * *
+     * * - +
+     * <p>
+     * Note that 21 % 3 == 0 and 21 % 7 = 0, but output is not +-, but +
+     */
+    public static void printMatrix() {
+        Scanner scanner = new Scanner(System.in);
+        Random random = new Random();
+        int row = scanner.nextInt();
+        int column = scanner.nextInt();
+        int[][] twoDArray = new int[row][column];
+        String[][] twoDArray_Str = new String[row][column];
+        for (int i = 0; i < twoDArray.length; i++) {
+            for (int j = 0; j < twoDArray[i].length; j++) {
+                twoDArray[i][j] = random.nextInt(100);
+                if (twoDArray[i][j] % 3 == 0) {
+                    twoDArray_Str[i][j] = "+";
+                } else if (twoDArray[i][j] % 7 == 0) {
+                    twoDArray_Str[i][j] = "-";
+                } else {
+                    twoDArray_Str[i][j] = "*";
+                }
+
+            }
+            System.out.println(Arrays.toString(twoDArray_Str[i]));
+        }
+    }
+
+    /**
+     * (optional)
+     * Method should print all prime numbers < 1000
+     */
+    public static void printPrimeNumbers() {
+        StringBuilder str = new StringBuilder();
+        boolean isPrime = true;
+        for (int i = 1; i < 1000; i++) {
+            for (int j = 2; j < i; j++) {
+                if (i % j == 0) {
+                    isPrime = false;
+                    break;
+                } else {
+                    isPrime = true;
+                }
+            }
+            if (isPrime) {
+                str.append(i).append(',');
```

Comment:
using StringBuilder for practice is great, but in that exact case, I'd say that this is redundant. Instead of storing a String with all prime numbers, you could just call System.out.print(or println)(i + ",") at line 113